### PR TITLE
Add support for collapsible fieldsets that start expanded

### LINKED ISSUE #173: Add fieldsets collapsed default value option.
**Ideas:**
- Default fieldsets collapsed value configured in `settings`.
- Default fieldsets collapsed value configured in the admin `Theme` options.
- Strictly Related to #160

### PR DESCRIPTION
**Pull request by merwok**
fixes #173

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,11 @@ django-admin-interface is a modern **responsive flat admin interface customizabl
 - Environment name/marker
 - Language chooser
 - List filter dropdown
-- `NEW` **Foldable apps** *(accordions in the navigation bar)*
-- `NEW` **List filter sticky**
-- `NEW` **Form controls sticky** *(pagination and save/delete buttons)*
-- Compatibility / Style optimizations for:
+- Foldable apps *(accordions in the navigation bar)*
+- Collapsible fieldsets can have their initial state be expanded
```
Comment (fabiocaccamo) [inline README.md:32]: Maybe adding here a link to the relative admin section would be useful.

File: README.md [hunk 2]
```diff
@@ -65,6 +66,18 @@ SILENCED_SYSTEM_CHECKS = ["security.W019"]
 - Run `python manage.py collectstatic`
 - Restart your application server
 
+#### Optional features
+
+To make a fieldset start expanded with a `Hide` button to collapse:
+```python
+    fieldsets = [
+        ("Section title", {
+            "classes": ("collapse", "initial-show"),
```
Comment (fabiocaccamo) [inline README.md:75]: I think that a class name like `collapse-expanded` or just `expanded` would be more intuitive and easier to remember.

**Review by fabiocaccamo — CHANGES_REQUESTED** [review]

**Comment by merwok** [comment]
Because there is a separate ticket about remembering the collapsed state, and I have to spend time on the list filter changes for now, I am marking this ready for review on its own!

**Review by fabiocaccamo — APPROVED** [review]
Thank you very much for this PR!

**Pull request merged**